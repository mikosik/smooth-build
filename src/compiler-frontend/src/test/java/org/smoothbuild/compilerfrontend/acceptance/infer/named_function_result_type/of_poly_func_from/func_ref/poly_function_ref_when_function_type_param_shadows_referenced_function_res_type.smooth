A myId(A a) = a;
myFunc(A a) = myId;