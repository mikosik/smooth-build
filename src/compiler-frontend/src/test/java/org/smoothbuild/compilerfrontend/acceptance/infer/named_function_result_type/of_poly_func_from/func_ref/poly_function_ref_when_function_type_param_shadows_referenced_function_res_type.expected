A myId<A>(A a)
  = a<>;
(B)->B myFunc<A,B>(A a)
  = myId<B>;
